xapian-core/tests/harness/backendmanager_remotetcp.cc

```diff
@@ -211,19 +216,21 @@ launch_xapian_tcpsrv(const string & args)
 
     // Find a slot to track the pid->fd mapping in.  If we can't find a slot
     // it just means we'll leak the fd, so don't worry about that too much.
-    for (unsigned i = 0; i < sizeof(pid_to_fd) / sizeof(pid_fd); ++i) {
+	unsigned pididx = 0;
+	for (unsigned i = 0; i < sizeof(pid_to_fd) / sizeof(pid_fd); ++i) {
 	if (pid_to_fd[i].pid == 0) {
-	    pid_to_fd[i].fd = tracked_fd;
-	    pid_to_fd[i].pid = child;
-	    break;
+		pid_to_fd[i].fd = tracked_fd;
+		pid_to_fd[i].pid = child;
+		pididx = i;
+		break;
```

Comment:
You've reindented the above and now it no longer conforms with our coding conventions.